### PR TITLE
[@types/k6] Add experimental browser module type annotations for version v0.44.0

### PR DESCRIPTION
**Pull request by ankur22**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>> not published yet
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by mstoykov — APPROVED** [review]
LGTM in the parts that aren't the exact definitions - where I am likely not the best person to review

File: types/k6/experimental/browser.d.ts [hunk 1]
```diff
@@ -0,0 +1,1508 @@
...
+   */
+  goto(url: string, options?: {
+    /**
+     * Referer header value.
+     */
+    referer?: string;
+
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```
Comment (ka3de) [inline types/k6/experimental/browser.d.ts:435]: I believe this `navigationTimeout` option in the config does not apply to our implementation right?

File: types/k6/experimental/browser.d.ts [hunk 2]
```diff
@@ -0,0 +1,1508 @@
...
+
+  /**
+   * This reloads the current page Returns the main resource response.
+   *
+   * @param options
+   */
+  reload(options?: {
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```
Comment (ka3de) [inline types/k6/experimental/browser.d.ts:844]: I believe this `navigationTimeout` option in the config does not apply to our implementation right?

File: types/k6/experimental/browser.d.ts [hunk 3]
```diff
@@ -0,0 +1,1508 @@
...
+  /**
+   * Set the supplied html string to the current page.
+   *
+   * @param html HTML markup to assign to the page.
+   * @param options
+   */
+  setContent(html: string, options?: {
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```
Comment (ka3de) [inline types/k6/experimental/browser.d.ts:980]: I believe this `navigationTimeout` option in the config does not apply to our implementation right?

File: types/k6/experimental/browser.d.ts [hunk 4]
```diff
@@ -0,0 +1,1508 @@
...
+   * - `'networkidle'` - **DISCOURAGED** consider operation to be finished
+   * when there are no network connections for at least `500` ms. Don't use
+   * this method for testing especially with chatty websites where the event
+   * may never fire, rely on web assertions to assess readiness instead.
+   * @param options
+   */
+  waitForLoadState(state?: "load"|"domcontentloaded"|"networkidle", options?: {
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```
Comment (ka3de) [inline types/k6/experimental/browser.d.ts:1298]: I believe this `navigationTimeout` option in the config does not apply to our implementation right?

File: types/k6/experimental/browser.d.ts [hunk 5]
```diff
@@ -0,0 +1,1508 @@
...
+  /**
+   * Waits for the given navigation lifecycle event to occur and returns the main
+   * resource response.
+   *
+   * @param options
+   */
+  waitForNavigation(options?: {
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```
Comment (ka3de) [inline types/k6/experimental/browser.d.ts:1316]: I believe this `navigationTimeout` option in the config does not apply to our implementation right?

**Review by ka3de — COMMENTED** [review]
Great job @ankur! :clap: 
I just made a couple of comments.

File: types/k6/experimental/browser.d.ts [hunk 6]
```diff
@@ -0,0 +1,1508 @@
+/**
+ * This module provides an experimental implementation that brings browser
+ * test automation to the k6 testing platform.
+ */
+
+/**
+ * Page provides methods to interact with a single tab in a browser in Chromium.
```
[the remark this answers is not in the document]
Comment (ankur22) [reply types/k6/experimental/browser.d.ts]: Yeah, happy to make that change. Done in dfc673054545f7efcd3e844a1cec222d435ed1fa

File: types/k6/experimental/browser.d.ts (hunk 5 above)
Comment (ankur22) [reply types/k6/experimental/browser.d.ts]: Correct. I've updated this and the timeout from `0` to `30` seconds in 988eda2e2e4243f27bd7ec84d52dc6206cb25402.

**Review by ka3de — APPROVED** [review]
LGTM.

**Review by mstoykov — APPROVED** [review]

**Comment by mstoykov** [comment]
Ready to merge

**Pull request merged**